core/imt/inc/ROOT/RTaskArena.hxx

```diff
@@ -0,0 +1,79 @@
+// @(#)root/thread:$Id$
```

Comment:
Does this line do anything or is it auto-generated by your editor?

Reply:
This is in all the other imt headers, I just copied the copyright statement adapting the author and the date. Not really sure of what it does or if It's necessary 🤷‍♂️

Reply:
you can ask Axel, I think it was used by the old homegrown reference guide generator. I don't think it's a doxygen thing, and it could be removed

Reply:
@Axel-Naumann Is this still needed?

Reply:
"It is not needed anymore. It's nice to keep it, to signal which module a file (esp a header, think include/) is in."